h2o-r/h2o-package/R/connection.R

```diff
@@ -348,7 +348,7 @@ h2o.connect <- function(ip = "localhost", port = 54321, strict_version_check = T
 h2o.getConnection <- function() {
   conn <- .attemptConnection()
   if (is.null(conn))
-    stop("No active connection to an H2O cluster. Did you run `h2o.init()` ?")
```

Comment:
I would remove the whole `Did you run h2o.init()?` sentence. It makes no sense to end up with "Did you run `` ?"

Reply:
It will always be either with h2o.init() or h2o.connect(). SUBST_START_DOCUMENTATION is replaced with ""

Reply:
My mistake!